C/services/filter-plugin-interfaces/python/filter_ingest_pymodule/ingest_callback_pymodule.cpp

```diff
@@ -125,11 +125,13 @@ void filter_plugin_ingest_fn(PyObject *ingest_callback,
 		return;
 	}
 
+#if 0
```

Comment:
Let's remove if not needed or use a meaningful symbolic name in the #if it there are circumstances that you might wish to turn it on